src/utils/lambda/event-source.ts

```diff
@@ -0,0 +1,80 @@
+import type { IEventSourceDlq } from "@aws-cdk/aws-lambda";
```

Comment:
I'll need to move some [existing code](https://github.com/guardian/cdk/tree/d1002d06d8d7f8bcae963e0485033d654cb3e094/src/utils) into more specific subdirectories in order to match this new structure. I will do that in its own PR as it's not directly related to this pattern.

Reply:
https://github.com/guardian/cdk/pull/314